roles/from_drive_list_lv/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+
+- name: 'normalize drive name'
```

Comment:
In the deployment part of all LVM stuff, we require the user to configure the full drive path (`/dev/sdb` etc). Not sure it's worth 'supporting' something different in this playbook.

Reply:
but here it is shell interaction (and not a configuration file), it's mainly to avoid typo on '/dev/'. But If you think it's better to remove it. Just reply